src/test_workflow/integ_test/integ_test_suite_opensearch_dashboards.py

```diff
@@ -81,6 +94,38 @@ def __setup_cluster_and_execute_test_config(self, config: str) -> int:
             os.chdir(self.work_dir)
             return self.execute_integtest_sh(endpoint, port, security, config)
 
+    def multi_execute_integtest_sh(self, cluster_endpoints: list, security: bool, test_config: str) -> int:
+        script = ScriptFinder.find_integ_test_script(self.component.name, self.repo.working_directory)
+
+        def custom_node_endpoint_encoder(node_endpoint: NodeEndpoint) -> dict:
+            return {"endpoint": node_endpoint.endpoint, "port": node_endpoint.port, "transport": node_endpoint.transport}
+        if os.path.exists(script):
+            single_data_node = cluster_endpoints[0].data_nodes[0]
```

Comment:
Naming can be better since there is no so-called "data-node" in OSD setups.

Reply:
I was expecting this is form the cluster_endpoints properties https://github.com/opensearch-project/opensearch-build/blob/59ab4340d1e788d84c6058c04e4ea75f86a29fa8/src/test_workflow/integ_test/topology.py#L186

Reply:
Done